[doc][broker][monitoring] add message_ack_rate metric doc (#15790)
apache/pulsar: site2/docs/reference-metrics.md

| pulsar_rate_out | Gauge | The total message rate of the namespace going out from this broker (message per second). |
| pulsar_throughput_in | Gauge | The total throughput of the namespace coming into this broker (byte per second). |
| pulsar_throughput_out | Gauge | The total throughput of the namespace going out from this broker (byte per second). |

> | pulsar_consumer_msg_ack_rate | Gauge | The total message acknowledge rate of the namespace coming into this broker  (message per second). |

Reviewer: ```suggestion
| pulsar_consumer_msg_ack_rate | Gauge | The total message acknowledgment rate of the namespace owned by this broker  (message per second). |
```

Reviewer: There are two spaces before `(`. Can you remove one?

| pulsar_publish_rate_limit_times | Gauge | The number of times the publish rate limit is triggered. |
| pulsar_throughput_in | Gauge | The total throughput of the topic coming into this broker (byte per second). |
| pulsar_throughput_out | Gauge | The total throughput of the topic going out from this broker (byte per second). |

> | pulsar_consumer_msg_ack_rate | Gauge | The total message acknowledge rate of the topic coming into this broker  (message per second). |

Reviewer: ```suggestion
| pulsar_consumer_msg_ack_rate | Gauge | The total message acknowledgment rate of the topic coming into this broker  (message per second). |
```

Reviewer: Is it a rate of a namespace or a topic? The subject is different between Line175 and Line224.

Reviewer: OK. My suggestion is to use "connected to" rather than "coming into", following the description of `pulsar_consumers_count`.
